relay-server/src/services/store.rs

```diff
@@ -199,6 +202,17 @@ impl StoreService {
             KafkaTopic::Attachments
         } else if event_item.as_ref().map(|x| x.ty()) == Some(&ItemType::Transaction) {
             KafkaTopic::Transactions
+        } else if event_item.as_ref().map(|x| x.ty()) == Some(&ItemType::UserReportV2) {
+            let feedback_ingest_topic_rollout_rate = self
+                .global_config
+                .current()
+                .options
+                .feedback_ingest_topic_rollout_rate;
```

Comment:
tiny nit:

```suggestion
            let global_config = self.global_config.current();
            let topic_rollout_rate = global_config.options.feedback_ingest_topic_rollout_rate;
```

I like splitting this, so it takes up less lines and becomes a bit more readable. But need to check if rustfmt aggress with me here ...